### ISSUE TITLE
Property appearance does not work

### ISSUE THREAD

**Issue by BorisBresciani** [postcss/autoprefixer#1098]
The property "appearance" does not work it stays just like this:
`appearance: none;`

Package.json
```
{
  "name": "test app",
  "version": "1.6.0",
  "private": true,
  "dependencies": {
    "bootstrap-scss": "^4.1.3",
    "moment": "^2.22.2",
    "moment-timezone": "^0.5.21",
    "react": "^16.4.2",
    "react-dom": "^16.4.2",
    "react-redux": "^5.0.7",
    "react-router-dom": "^4.3.1",
    "reactstrap": "^6.3.1",
    "redux": "^4.0.0",
    "redux-thunk": "^2.3.0",
    "whatwg-fetch": "^2.0.4"
  },
  "scripts": {
    "start": "parcel public/index.html -p 3000 --open",
    "build": "parcel build public/index.html --out-dir build/ --public-url ./"
  },
  "devDependencies": {
    "autoprefixer": "^9.1.2",
    "babel-eslint": "^8.2.6",
    "babel-plugin-transform-object-rest-spread": "^6.26.0",
    "babel-preset-react": "^6.24.1",
    "eslint": "^5.4.0",
    "eslint-plugin-react": "^7.11.1",
    "node-sass": "^4.9.3",
    "parcel-bundler": "^1.9.7",
    "postcss-modules": "^1.3.2",
    "sass": "^1.12.0"
  },
  "babel": {
    "presets": [
      "react"
    ],
    "plugins": [
      "transform-object-rest-spread"
    ]
  },
  "browserslist": [
    "> 1%",
    "last 5 versions",
    "not ie < 9"
  ],
  "postcss": {
    "plugins": {
      "autoprefixer": {
        "grid": true
      }
    }
  }
}

```

Thank you for your help

**Comment by ai** [comment]
Does other prefixes work? Try `user-select: none`

**Comment by BorisBresciani** [comment]
Yes with `user-select: none` this is transformed but not with  `appearance: none;`
This is not due to the fact that the css that contains "appearance: none" is external? (It is in node_modules)

**Comment by ai** [comment]
Call `npx autoprefixer --info` and post it's output

**Comment by BorisBresciani** [comment]
```
Browsers:
  Chrome for Android: 67
  Firefox for Android: 60
  And_qq: 1.2
  UC for Android: 11.8
  Android: 67, 4.4.3-4.4.4, 4.4, 4.2-4.3, 4.1
  Baidu: 7.12
  Bb: 10, 7
  Chrome: 68, 67, 66, 65, 64
  Edge: 17, 16, 15, 14, 13
  Firefox: 61, 60, 59, 58, 57
  IE: 11, 10, 9
  IE Mobile: 11, 10
  iOS: 11.3-11.4, 11.0-11.2, 10.3, 10.0-10.2, 9.3
  Opera Mini: all
  Opera Mobile: 46, 12.1, 12, 11.5, 11.1
  Opera: 53, 52, 51, 50, 49
  Safari: 11.1, 11, 10.1, 10, 9.1
  Samsung: 7.2, 6.2, 5, 4

These browsers account for 90.17% of all users globally

At-Rules:
  @keyframes: webkit
  @viewport: ms, o
  @resolution: webkit, o

Selectors:
  ::selection: moz
  ::placeholder: webkit, -ms- old, ms
  :fullscreen: webkit, moz, ms
  ::backdrop: webkit
  :read-only: moz
  :read-write: moz
  :any-link: webkit, moz

Properties:
  box-shadow: webkit
  animation: webkit
  animation-name: webkit
  animation-duration: webkit
  animation-delay: webkit
  animation-direction: webkit
  animation-fill-mode: webkit
  animation-iteration-count: webkit
  animation-play-state: webkit
  animation-timing-function: webkit
  transition: webkit, o
  transition-property: webkit, o
  transition-duration: webkit, o
  transition-delay: webkit, o
  transition-timing-function: webkit, o
  transform: webkit, ms
  transform-origin: webkit, ms
  perspective: webkit
  perspective-origin: webkit
  transform-style: webkit
  backface-visibility: webkit
  border-image: webkit, o
  mask: webkit
  mask-image: webkit
  box-sizing: webkit
  filter: webkit
  backdrop-filter: webkit
  columns: webkit
  column-width: webkit
  column-gap: webkit
  column-rule: webkit
  column-rule-color: webkit
  column-rule-width: webkit
  column-count: webkit
  column-rule-style: webkit
  column-span: webkit
  column-fill: webkit
  break-before: webkit
  break-after: webkit
  break-inside: webkit
  user-select: webkit, moz, ms
  flex: -webkit- 2009, ms
  flex-grow: -webkit- 2009, ms
  flex-shrink: -webkit- 2009, ms
  flex-basis: -webkit- 2009, ms
  flex-direction: -webkit- 2009, ms
  flex-wrap: -webkit- 2009, ms
  flex-flow: -webkit- 2009, ms
  justify-content: -webkit- 2009, ms
  order: -webkit- 2009, ms
  align-items: -webkit- 2009, ms
  align-self: -webkit- 2009, ms
  align-content: -webkit- 2009, ms
  background-clip: webkit
  font-feature-settings: webkit
  font-variant-ligatures: webkit
  font-language-override: webkit
  font-kerning: webkit
  hyphens: webkit, ms
  tab-size: moz, o
  grid-template *: ms
  grid-template-rows *: ms
  grid-template-columns *: ms
  touch-action: ms
  text-decoration-style: webkit
  text-decoration-color: webkit
  text-decoration-line: webkit
  text-decoration: webkit
  text-decoration-skip: webkit
  text-size-adjust: webkit, moz, ms
  mask-clip: webkit
  mask-composite: webkit
  mask-origin: webkit
  mask-repeat: webkit
  mask-border-repeat: webkit
  mask-border-source: webkit
  mask-position: webkit
  mask-size: webkit
  mask-border: webkit
  mask-border-outset: webkit
  mask-border-width: webkit
  mask-border-slice: webkit
  clip-path: webkit
  box-decoration-break: webkit
  object-fit: o
  object-position: o
  shape-margin: webkit
  shape-outside: webkit
  shape-image-threshold: webkit
  text-overflow: o
  image-rendering: ms
  border-inline-start: webkit
  border-inline-end: webkit
  margin-inline-start: webkit
  margin-inline-end: webkit
  padding-inline-start: webkit
  padding-inline-end: webkit
  border-block-start: webkit
  border-block-end: webkit
  margin-block-start: webkit
  margin-block-end: webkit
  padding-block-start: webkit
  padding-block-end: webkit
  appearance: webkit, moz
  scroll-snap-type: webkit, ms
  scroll-snap-coordinate: webkit, ms
  scroll-snap-destination: webkit, ms
  scroll-snap-points-x: webkit, ms
  scroll-snap-points-y: webkit, ms
  flow-into: webkit, ms
  flow-from: webkit, ms
  region-fragment: webkit, ms
  writing-mode: webkit, ms
  text-emphasis: webkit
  text-emphasis-position: webkit
  text-emphasis-style: webkit
  text-emphasis-color: webkit
  grid-row-start *: ms
  grid-column-start *: ms
  grid-row-end *: ms
  grid-column-end *: ms
  grid-row *: ms
  grid-column *: ms
  grid-area *: ms
  grid-template-areas *: ms
  grid-column-align *: ms
  grid-row-align *: ms
  text-spacing: ms
  overscroll-behavior: ms
  color-adjust: webkit

Values:
  linear-gradient: -webkit- old, webkit, o
  repeating-linear-gradient: -webkit- old, webkit, o
  radial-gradient: -webkit- old, webkit, o
  repeating-radial-gradient: -webkit- old, webkit, o
  filter: webkit
  element: moz
  image-set: webkit
  cross-fade: webkit
  flex: -webkit- 2009, ms
  inline-flex: -webkit- 2009, ms
  grid *: ms
  inline-grid *: ms
  max-content: webkit, moz
  min-content: webkit, moz
  fit-content: webkit, moz
  fill: webkit, moz
  fill-available: webkit, moz
  stretch: webkit, moz
  zoom-in: webkit
  zoom-out: webkit
  grab: webkit
  grabbing: webkit
  sticky: webkit
  pixelated: webkit, moz, o
  isolate: webkit

* - Prefixes will be added only on grid: true option.

```

**Comment by ai** [comment]
Hm, `appearance` with the right prefixes are in `--info`. Also, we have [simple test for this prefix](https://github.com/postcss/autoprefixer/blob/master/test/cases/appearance.out.css) and I tested it with fresh Can I Use data.

Let’s check the most common problems:

1. Maybe there is a typo in CSS property?
2. Maybe there is a Autoprefixer’s [control comments](https://github.com/postcss/autoprefixer#control-comments) somewhere?
3. Maybe there is some additional tool in Parcel, which cut prefixes? Can you install `postcss-cli` and test CSS file by CLI without any other build tool?

If we will not find problems here, you should create a small test project to reproduce the problem.

**Comment by ai** [comment]
BTW, because of your issue, we found small UX bug. Now `--info` output is sorted https://github.com/postcss/autoprefixer/commit/6dae47e93a9abf9daad1790bc57d5567a2b76197

**Comment by ai** [comment]
Glad to hear that the problem was fixed :D

**Comment by ai** [comment]
`--info` sorting was released in 9.1.3
